src/components/cookies/CookieConsent.tsx

```diff
@@ -34,6 +36,17 @@ export const CookieConsent = () => {
         j.src = "https://www.googletagmanager.com/gtm.js?id=" + i + dl;
         f.parentNode.insertBefore(j, f);
       })(window, document, "script", "dataLayer", "GTM-NTNH983");
+      // Matomo
+      (function () {
+        var _mtm = (window._mtm = window._mtm || []);
+        _mtm.push({ "mtm.startTime": new Date().getTime(), event: "mtm.Start" });
+        var d = document,
+          g = d.createElement("script"),
+          s = d.getElementsByTagName("script")[0];
+        g.async = true;
+        g.src = "https://cdn.matomo.cloud/climatelaws.matomo.cloud/container_l8tXApwR.js";
```

Comment:
Question - do we want to pull out the google id "GTM-NTNH983" and the specific js file for matamo into config?